python/ray/autoscaler/kuberay/ray-cluster.complete.yaml

```diff
@@ -22,15 +22,22 @@ spec:
     # rayNodeType: head # Not needed since it is under the headgroup
     # the following params are used to complete the ray start: ray start --head --block --redis-port=6379 ...
     rayStartParams:
+      # Flag "no-monitor" must be set when running the autoscaler in
+      # a sidecar container.
+      no-monitor: "true"
```

Comment:
Can we run autoscaler in a sidecar container currently?  Maybe we need to update the Raycluster CRD too.

Reply:
This example CR sets up the autoscaler to run in a sidecar.

In future work, we should update the operator and custom resource to automatically configure the autoscaler sidecar
(or conceivably an independent autoscaler deployment, if it's desired).